numbat/src/quantity.rs

```diff
@@ -24,20 +24,31 @@ pub type Result<T> = std::result::Result<T, QuantityError>;
 pub struct Quantity {
     value: Number,
     unit: Unit,
+    can_simplify: bool,
```

Comment:
The name confused me a bit when first reading this code. Maybe call it `simplification_allowed`? And rename `no_simplify` to `prevent_simplification`?